src/components/adminBar/TheAposAdminBar.vue

```diff
@@ -34,20 +34,23 @@
         />
       </li>
     </ul>
+    <TheAposAdminBarUser class="apos-admin-bar__user" :user="user" />
   </nav>
 </template>
 
 <script>
 import AposLogo from './AposLogo';
 import AposButton from '../button/AposButton';
+import TheAposAdminBarUser from './TheAposAdminBarUser';
 import AposContextMenu from '../contextMenu/AposContextMenu';
 
 export default {
   name: 'TheAposAdminBar',
   components: {
     AposLogo,
     AposButton,
-    AposContextMenu
+    AposContextMenu,
+    TheAposAdminBarUser
```

Comment:
what the what

Reply:
The `The` thing? Because it's only ever used once.

Reply:
is that a real pattern? seems horrible for scanning dirs alphabetically

Reply:
From here: https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended